pkg/telemetry/telemetry.go

```diff
@@ -35,6 +32,10 @@ func (t Telemetry) L() *zap.Logger {
 func DefaultConfig() *Config {
 	return &Config{
 		LogConfig: DefaultLogConfig(),
+		Resource: map[string]string{
+			string(semconv.ServiceNameKey):       "glide",
+			string(semconv.ServiceInstanceIDKey): uuid.New().String(),
```

Comment:
Can we check if there is the `POD_NAME` env var to give it precedence before falling back for UUID? 
It seems to be the closes value in the K8s world to the meaning of `service.instance.id`

Reply:
An issue on the chart repo side: https://github.com/EinStack/charts/issues/12

Reply:
done eb8506f

Reply:
This is great, thanks you for the update! I have realized that users had this opportunity to pull POD_NAME even without the update, because glide config allows passing env variables like `${env:POD_NAME}` as config values. But I guess this is one less thing for the end users to think about.